test/runnable/aliasthis.d

```diff
@@ -1268,11 +1268,11 @@ void test8735()
     // 9709 case
     alias A = Tuple9709!(1,int,"foo");
     A a;
-    static assert(A[0] == 1);
+    //static assert(A[0] == 1);
     static assert(a[0] == 1);
     //static assert(is(A[1] == int));
     //static assert(is(a[1] == int));
-    static assert(A[2] == "foo");
+    //static assert(A[2] == "foo");
```

Comment:
I don't remember the precise reason, but it had been interfere with the new operator overloading rule. And the test case was added without deep thinking when issue 8735 was fixed (actually it was added by *me*).

So I was disable it again because it would not introduce serious use code breaking.